bigbluebutton-html5/imports/ui/components/modal/fullscreen/styles.scss

```diff
@@ -33,6 +33,10 @@
   flex: 1;
   margin: 0;
   font-weight: 400;
+
+  @include mq($small-only) {
+    font-size: x-large;
```

Comment:
1. We use 'rem' as font-size units everywhere so that we could change the font-size of the root (Settings -> Application) and it was changed in the whole application.
2. It is said that this spec is implemented differently in different browsers, thus it's not recommended to change font-sizes this way.
3. It doesn't fix the overflow problem, try to increase the font-size in the settings.